src/Controllers/DatabaseMock.java

```diff
@@ -4,10 +4,10 @@
 import java.util.List;
 import model.*;
 public class DatabaseMock {
-    int incrementKey = 1;
-    private final List<model.StudentModel> students = new ArrayList<>();
+    private final List<StudentModel> students = new ArrayList<>();
     private final List<TeacherModel> teachers = new ArrayList<>();
 
+
     public void getAllStudents() {
```

Comment:
When you call a method get... it is suppose to return some value.
Here the correct method name would be printAllStudents() but this is not something that the database layer should take care of. Best approach here would be to return the information and print it where you are calling the method.

The same applies for all the other methods.